app/libs/soundcloud.js

```diff
@@ -134,6 +241,42 @@ class Soundcloud {
             success: callback
         });
     }
+
+    savedNotification(values) {
+        let notification;
+        if (values.length == 1) {
+            notification = values[0];
+            application.channel.request('notification', notification);
+        } else {
+            notification = {
+                status: 'ok',
+                message: t('stream track imported')
+            }
+
+            for (let i = 0; i < values.length; i++) {
+                let el = values[i];
+                if (el.status == 'ko') {
+                    notification = {
+                        status: 'ko',
+                        message: t('one track not imported')
+                    }
+                }
+                break;
```

Comment:
the break is used to break out of the for loop if as soon as an `el.status == 'ko'`.